src/scss/main.scss

```diff
@@ -250,9 +250,59 @@ button:hover {
   text-align: left;
   padding: 1em;
   background-color: white;
-  
+
+}
+
+#bodyWelcome {
+  background-image: none;
+  background: rgb(165, 180, 255);
+  background: linear-gradient(90deg, rgb(170, 182, 241) 0%, rgb(171, 178, 240) 49%, rgb(164, 161, 245) 100%);
+}
```

Comment:
We can make sure to use the features of SCCS here. Please make a file in the folder `pages` and call it `welcome.scss` file. Then import that file in `index.scss`, just like the others.

Inside this file we'll add file our identifier like you've done with the `id` of `bodyWelcome` (Note that we could also a class, but an id is fine). 
```scss
#bodyWelcome {
  // add your other elements here
  .welcomeHeader {
    // like so
  }
  // And more here
}
```